mysql-test/suite/rocksdb/t/iterator_bounds.test

```diff
@@ -0,0 +1,27 @@
+#
+# Issue #878: Descending scans from reverse column families return no results
+# due to iterator bounds
+#
+
+create table t (i int primary key, j int, key(j) comment 'rev:bf5_2');
```

Comment:
I assume your tree is correct in this regard (although I was not able to find --default-storage-engine=rocksdb for this MTR suite in all of 10 seconds of searching), but at least our downstream would appreciate explicit engine=rocksdb clauses.

Reply:
We add the option when we run mtr. See Sergey's comment:
https://github.com/facebook/mysql-5.6/pull/868#issuecomment-413220539